### PR TITLE
Color variables

### PR DESCRIPTION
**Pull request by ejanzer**
Moving all color variable definitions to a single .scss file and using `@import` to use those variables everywhere else. I (mostly) used [the palette in Figma](https://www.figma.com/file/GmfndRmQChBeoklOkmi0FC/Open-Disclosure?node-id=2029%3A0), but there were a few things missing that I left in and made up a name for.

Also cleaning up some of the places where we still use hex values in our .scss files - I mostly just used the variable with the same hex value, except for the places where 'black' or #000 was used, which I swapped for $primaryBlack. I left the .css files alone.

### CODE REVIEW

File: src/styles/_colors.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+$primaryOffWhite: #fafafa;
+$primaryWhite: #ffffff;
+
+// Secondary palette
+$secondaryGreen: #e3f9f1;
+$secondaryYellow: #feefd0;
+$secondaryBlue: #deeffe;
+$secondaryGray: #f2f2f2;
+$darkGray: #555555;
+$secondaryWhite: #e5e5e5;
```
Comment (ejanzer) [inline src/styles/_colors.scss:13]: This is the page background color in most of the mocks in Figma, but it doesn't appear in the palettes under Design System. I'm not sure if this is actually supposed to be $primaryWhite, or if we need to keep it separate.

Comment (JMStudiosJoe) [reply src/styles/_colors.scss:13]: Need to update or ready for merge?

Comment (ejanzer) [reply src/styles/_colors.scss:13]: @JMStudiosJoe Ok to merge, I was just curious if anyone knew whether that color should be used as primary white, or if it's only for backgrounds. But I can always change it later if needed.

Comment (alessandro-pianetta) [reply src/styles/_colors.scss:13]: The Design System section on Figma defines primary white as #FFFFFF, so I think we're good on that front!

**Pull request merged**